tools/mo/openvino/tools/mo/utils/cli_parser.py

```diff
@@ -1490,15 +1490,15 @@ def split_inputs(input_str):
 
 def split_shapes(argv_input_shape: str):
     range_reg = r'([0-9]*\.\.[0-9]*)'
-    first_digit_reg = r'([0-9 ]+|-1|\?|{})'.format(range_reg)
+    first_digit_reg = r'([0-9 ]*|-1|\?|{})'.format(range_reg)
     next_digits_reg = r'(,{})*'.format(first_digit_reg)
     tuple_reg = r'((\({}{}\))|(\[{}{}\]))'.format(first_digit_reg, next_digits_reg,
                                                   first_digit_reg, next_digits_reg)
 
     full_reg = r'^{}(\s*,\s*{})*$|^$'.format(tuple_reg, tuple_reg)
     if not re.match(full_reg, argv_input_shape):
         raise Error('Input shape "{}" cannot be parsed. ' + refer_to_faq_msg(57), argv_input_shape)
-    return re.findall(r'[(\[]([0-9,\.\? -]+)[)\]]', argv_input_shape)
+    return re.findall(r'[(\[]([0-9,\.\? -]*)[)\]]', argv_input_shape)
```

Comment:
Do we need convert_model() to support following syntax?
convert_model(input_shape=[])
convert_model(input=InputCutInfo(shape=[]))

Reply:
`convert_model` is already supporting inputs with scalar shapes. Added an test case for that